plugins/source/alicloud/resources/services/bss/fetch.go

```diff
@@ -0,0 +1,174 @@
+package bss
+
+import (
+	"context"
+	"fmt"
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/bssopenapi"
+	"github.com/cloudquery/cloudquery/plugins/source/alicloud/client"
+	"github.com/cloudquery/cloudquery/plugins/source/alicloud/codegen/models"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"net/http"
+	"time"
+)
+
+var (
+	_maxLimit = 100
+)
+
+// See: https://help.aliyun.com/document_detail/100400.html
+func fetchBssBillOverview(_ context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan<- any) error {
+	spec := meta.(*client.Client).Spec
+	cli, err := bssopenapi.NewClientWithAccessKey(spec.RegionID, spec.AccessKey, spec.SecretKey)
+	if err != nil {
+		return err
+	}
+
+	billingCycles := GetBillingCycles()
+	if spec.BillHistory > 0 {
+		billingCycles = append(GetHistoryBillingCycles(spec.BillHistory), billingCycles...)
+	}
+	for _, billingCycle := range billingCycles {
+		request := bssopenapi.CreateQueryBillOverviewRequest()
+		request.BillingCycle = billingCycle
+		response, err := cli.QueryBillOverview(request)
+		if err != nil {
+			return err
+		}
+		if response.GetHttpStatus() != http.StatusOK {
+			return fmt.Errorf("httpcode %d", response.GetHttpStatus())
+		}
+		for _, item := range response.Data.Items.Item {
+			res <- ToBSSBillOverview(response.Data.BillingCycle, response.Data.AccountID, response.Data.AccountName, item)
+		}
+	}
+	return nil
+}
+
+func ToBSSBillOverview(billingCycle, accountId, accountName string, item bssopenapi.Item) *models.BillOverview {
+	return &models.BillOverview{
+		BillingCycle:          billingCycle,
+		AccountID:             accountId,
+		AccountName:           accountName,
+		DeductedByCoupons:     item.DeductedByCoupons,
+		RoundDownDiscount:     item.RoundDownDiscount,
+		ProductName:           item.ProductName,
+		ProductDetail:         item.ProductDetail,
+		ProductCode:           item.ProductCode,
+		BillAccountID:         item.BillAccountID,
+		ProductType:           item.ProductType,
+		DeductedByCashCoupons: item.DeductedByCashCoupons,
+		OutstandingAmount:     item.OutstandingAmount,
+		BizType:               item.BizType,
+		PaymentAmount:         item.PaymentAmount,
+		PipCode:               item.PipCode,
+		DeductedByPrepaidCard: item.DeductedByPrepaidCard,
+		InvoiceDiscount:       item.InvoiceDiscount,
+		Item:                  item.Item,
+		SubscriptionType:      item.SubscriptionType,
+		PretaxGrossAmount:     item.PretaxGrossAmount,
+		PretaxAmount:          item.PretaxAmount,
+		OwnerID:               item.OwnerID,
+		Currency:              item.Currency,
+		CommodityCode:         item.CommodityCode,
+		BillAccountName:       item.BillAccountName,
+		AdjustAmount:          item.AdjustAmount,
+		CashAmount:            item.CashAmount,
+	}
+}
+
+/*
+* https://help.aliyun.com/document_detail/100392.html
```

Comment:
These links are great - we should also add them to the `Description` field of each table so they show up in the docs :)